### PR TITLE
isolation doc and dedicated configuration guide

### PR DESCRIPTION
**Pull request by corkrean**

### CODE REVIEW

File: docs/spicedb-dedicated/dedicated-configuration.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+<img src={require("/img/ps-type.png").default} alt="permission system type" />
+
+Please select the choice (Production or Development) that is appropriate for the Permission System you are deploying. This choice will not effect the performance or security of your Permission System, but may have impact your Permission System in the future, so please choose appropriately.
+
+### Datastore
+
+<img src={require("/img/datastore-config.png").default} alt="datastore configuration" />
+
+Your SpiceDB Dedicated environment can be provisioned with multiple isolated databases. If you have more than one, choose the database that is appropriate for the Permission System you are deploying.
```
Comment (dsieczko) [inline docs/spicedb-dedicated/dedicated-configuration.md:17]: You use Datastore to introduce the section, but then "database"

File: docs/spicedb-dedicated/dedicated-configuration.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+<img src={require("/img/channels-config.png").default} alt="channels config" />
+
+SpiceDB Dedicated has two update channels: `rapid` and `regular` . You can select the update channel either when you launch the Permission System or in the settings page after you’ve launched it.
+
+`rapid` - gets every release that is not a release candidate.
+
+`regular` - trails behind `rapid` by at least one release.
+
+After you’ve deployed your Permission System, you can choose to keep up to date with a channel automatically, or you can “pin” to a version by un-checking the automatic updates box. If you uncheck the automatic release box, you’ll still be able to update, but will have to pick the next version manually.
```
Comment (dsieczko) [inline docs/spicedb-dedicated/dedicated-configuration.md:29]: Can we add a screenshot of the box?

File: docs/spicedb-dedicated/dedicated-configuration.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+If you are deploying a single region Permission System, define a single cluster.
+
+#### Multi-region deployment
+
+Define a cluster for every region you want to deploy into.
+
+#### Cluster Class
+
+For production clusters, we recommend selecting the Cluster Class with the largest number of replicas and processing units. For non-production clusters, you can select whatever Cluster Class is appropriate for your workload. If you have questions about your overall utilization or would like to provision a custom Cluster Class, please reach out to your Authzed account team.
```
Comment (dsieczko) [inline docs/spicedb-dedicated/dedicated-configuration.md:54]: Auth"Z"ed

File: docs/spicedb-dedicated/dedicated-isolation.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# SpiceDB Dedicated Isolation
+
+SpiceDB Dedicated isolates your workloads from other customers. Cloud accounts, compute resources, databases, and networking are all dedicated to you.
+
+Additionally, SpiceDB allows you to deploy multiple isolated Permissions Systems into a single SpiceDB Dedicated environment. Each Permissions Systems has it’s own schema and set of relationships. Also, these Permissions Systems impose memory and CPU limits so one Permissions System can’t crowd out another Permissions System.
```
Comment (dsieczko) [inline docs/spicedb-dedicated/dedicated-isolation.md:5]: Additionally, SpiceDB *Dedicated allows

Comment (dsieczko) [reply docs/spicedb-dedicated/dedicated-isolation.md:5]: Typo in "Each Permissions System*s" should be "Permissions (or Permission pending Jimmy) System"

File: docs/spicedb-dedicated/dedicated-isolation.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+By default, API tokens are scoped to a particular Permissions System. [Fine Grained Access Management (FGAM)](/spicedb-dedicated/fgam.md) can take this farther by restricting API tokens to specified APIs, object types, or object IDs.
+
+The below diagram gives an overview of the SpiceDB Dedicated isolation model.
+
+<img src={require("/img/dedicated-isolation.png").default} alt="dedicated isolation model" />
```
Comment (dsieczko) [inline docs/spicedb-dedicated/dedicated-isolation.md:11]: The image here has mismatching font sizes and uses Authzed not AuthZed - the title "Cloud Account Operated by AuthZed and Dedicated to You" is kind of odd  and I don't think this makes it clear that it's in our Cloud organization. Also the Pod Icon should sit in top left and spill out of the container like the Kuberenetes icon.

**Review by dsieczko — CHANGES_REQUESTED** [review]

File: docs/spicedb-dedicated/dedicated-configuration.md (hunk 2 above)
Comment (corkrean) [reply docs/spicedb-dedicated/dedicated-configuration.md:29]: there already is a rapid vs regular box. i suspect you missed it?

File: docs/spicedb-dedicated/dedicated-configuration.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
+# SpiceDB Dedicated Configuration
+
+The guide provides information about the configuration options available to you when deploying a SpiceDB Dedicated Permission System.
```
Comment (dsieczko) [inline docs/spicedb-dedicated/dedicated-configuration.md:3]: This

File: static/img/dedicated_isolation_v2.png
Comment (dsieczko) [inline static/img/dedicated_isolation_v2.png:1]: So close! Looking better - Now just need to center the copy in the box - you're leaning right in a lot of the boxes. 

Also Authzed should be AuthZed.

You can also remove "Pod" and the label for CPU Limit and Memory Limit CPU is already called out in the unit and people will understand it's memory.

**Review by dsieczko — CHANGES_REQUESTED** [review]

File: docs/spicedb-dedicated/dedicated-configuration.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+If you are deploying a single region Permission System, define a single cluster.
+
+#### Multi-region deployment
+
+Define a cluster for every region you want to deploy into.
+
+#### Cluster Class
+
+For production clusters, we recommend selecting the Cluster Class with the largest number of replicas and processing units. For non-production clusters, you can select whatever Cluster Class is appropriate for your workload. If you have questions about your overall utilization or would like to provision a custom Cluster Class, please reach out to your AuthZed account team.
```
[the remark this answers is not in the document]
Comment (dsieczko) [reply docs/spicedb-dedicated/dedicated-configuration.md:56]: Sorry, should have been more explicit  [reach out to your AuthZed account team.](https://authzed.com/call)

**Review by dsieczko — CHANGES_REQUESTED** [review]

File: docs/spicedb-dedicated/dedicated-configuration.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+<img src={require("/img/datastore-config-v2.png").default} alt="datastore configuration" />
+
+Your SpiceDB Dedicated environment can be provisioned with multiple isolated [datastores](/spicedb/selecting-a-datastore.md). If you have more than one, choose the datastore that is appropriate for the Permissions System you are deploying.
+
+### Update Channel
+
+<img src={require("/img/channels-config.png").default} alt="channels config" />
+
+SpiceDB Dedicated has two update channels: `rapid` and `regular` . You can select the update channel either when you launch the Permissions System or in the settings page after you’ve launched it.
```
Comment (dsieczko) [inline docs/spicedb-dedicated/dedicated-configuration.md:23]: Don't need either here - up to you if you want to change

File: docs/spicedb-dedicated/dedicated-configuration.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (dsieczko) [reply docs/spicedb-dedicated/dedicated-configuration.md:56]: @corkrean  this one got hid in the resolved conversation - I just unresolved . See above

**Review by dsieczko — CHANGES_REQUESTED** [review]

**Review by dsieczko — CHANGES_REQUESTED** [review]

**Review by dsieczko — APPROVED** [review]
👍

**Pull request merged**